src/utils/common.js

```diff
@@ -12,4 +13,15 @@ function getContract(contractAddr, artifact) {
     return contract;
 }
 
+function getProviders() {
+    const env = require('dotenv').config().parsed
```

Comment:
This looks like server-side code? 
I think common.js runs on the client side and we should probably get the network information from metamask.
But yeah we should have a mapping of {network => contract address} somewhere. Probably ABI too if the versions are different, but until the contract finalizes on mainnet it's not that important.